File: python/1072.py
i=int(input())
ini=0
outi=0
for x in range(1,i+1):
    v=int(input())
    if v >= 10 and v <=20:
        ini+=1
    else:
        outi+=1
print(str(ini)+" in\n"+str(outi)+" out")

File: javascript/1072.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var a=parseInt(lines.shift())
var s;
var ini=0
var outi=0
for(let i=1;i<=a;i++){
    s=parseInt(lines.shift())
    if(s>=10 && s<=20){
        ini+=1;
    }else{
        outi+=1;
    };
}
console.log(`${ini} in\n${outi} out`)